eth/gasprice/gasprice_test.go

```diff
@@ -332,10 +330,50 @@ func TestSuggestTipCapMinGas(t *testing.T) {
 	}, defaultOracleConfig())
 }
 
-// Regression test to ensure that SuggestPrice does not panic with activation of Subnet EVM
+// Regression test to ensure that SuggestPrice does not panic prior to activation of Subnet EVM
 // Note: support for gas estimation without activated hard forks has been deprecated, but we still
 // ensure that the call does not panic.
-func TestSuggestGasPriceSubnetEVM(t *testing.T) {
+func TestSuggestGasPricePreSubnetEVM(t *testing.T) {
+	config := Config{
+		Blocks:     20,
+		Percentile: 60,
+	}
+
+	backend := newTestBackend(t, params.TestPreSubnetEVMConfig, 3, func(i int, b *core.BlockGen) {
+		b.SetCoinbase(common.Address{1})
+
+		signer := types.LatestSigner(params.TestPreSubnetEVMConfig)
+		gasPrice := big.NewInt(params.MinGasPrice)
+		for j := 0; j < 50; j++ {
+			tx := types.NewTx(&types.LegacyTx{
+				Nonce:    b.TxNonce(addr),
+				To:       &common.Address{},
+				Gas:      params.TxGas,
+				GasPrice: gasPrice,
+				Data:     []byte{},
+			})
+			tx, err := types.SignTx(tx, signer, key)
+			if err != nil {
+				t.Fatalf("failed to create tx: %s", err)
+			}
+			b.AddTx(tx)
+		}
+	})
+	oracle, err := NewOracle(backend, config)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = oracle.SuggestPrice(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+// Regression test to ensure that SuggestPrice does not panic prior to activation of SubnetEVM
+// Note: support for gas estimation without activated hard forks has been deprecated, but we still
+// ensure that the call does not panic.
+func TestSuggestGasPricePreAP3(t *testing.T) {
```

Comment:
AP3 should be renamed.

Reply:
I think we can completly get rid of this test. If we want to ensure `SuggestPrice` does not panic prior to Subnet-EVM activation, the test above does that. This one ensures if `SuggestPrice` panics after the Subnet-EVM activation (despite the test name). So we don't need to keep this `TestSuggestGasPricePreAP3` test at all.